mesonbuild/backend/ninjabackend.py

```diff
@@ -36,7 +36,7 @@
 from .. import compilers
 from ..arglist import CompilerArgs
 from ..compilers import Compiler
-from ..linkers import ArLikeLinker, RSPFileSyntax
+from ..linkers import ArLikeLinker, RSPFileSyntax, AppleArLinker
```

Comment:
Since the time this commit was originally introduced, we now check `static_linker.id == 'applear'` or similar logic. The hidden impl now causes an import error if you try to import it directly.

Reply:
fixed